### ISSUE TITLE
Traceback: While click on edit button

### ISSUE THREAD

**Issue by Ankit-Gandhi** [odoo/odoo#22916]
Impacted versions:
> OdooV11 Enterprise

Steps to reproduce:
Go to Settings > Technical > Actions > Reports > Select any report(Quotation / Order)
Change report from PDF to HTML
Now go to print that report screen and refresh that page
Try to edit that report
Current behavior:
> Got Odoo Client Error: Uncaught TypeError: Cannot read property 'indexOf' of undefined
Expected behavior:
> Report should be edit.

Video/Screenshot link (optional):
> Screen shot:
https://www.awesomescreenshot.com/image/3159679/ede70b96dc8b60497dc97277c0b928d4

**Comment by Yenthe666** [comment]
Hi @Ankit-Gandhi,

Thanks a lot for your bug report.
I've just tried this on the latest Odoo 11.0 (on runbot.odoo.com) but it seems to work fine:
![image](https://user-images.githubusercontent.com/6352350/35970171-ae8d8826-0cc9-11e8-9d2f-2a1632dbfdf8.png)

Can you update your Odoo source code and try again? Can you reproduce it on a runbot instance?

Regards,
Yenthe

**Comment by Ankit-Gandhi** [comment]
Hello @Yenthe666,

I have checked in runbot, Please reload print page and try to edit that page.

Best Thanks,
Ankit H Gandhi.

**Comment by Yenthe666** [comment]
Yup, but directly coming because of a side behaviour of https://github.com/odoo/odoo/issues/22913 really..

**Comment by abhi4u23** [comment]
[Yes](https://www.teamhindi.com/mirzapur-2-release-date/)
[Thanks](https://www.teamhindi.com/)
